src/transformers/modeling_fx_utils.py

```diff
@@ -144,6 +147,14 @@ class HFTracer(Tracer):
 
     def __init__(self, batch_size=1, sequence_length=[128, 128], num_choices=-1):
         super().__init__()
+
+        if not is_torch_fx_available():
+            torch_version = version.parse(importlib_metadata.version("torch"))
+            raise ImportError(
+                f"Found an incompatible version of torch. Found version {torch_version}, but only version "
+                f"{TORCH_FX_REQUIRED_VERSION} is supported."
```

Comment:
Maybe we can explicitly state that symbolic tracing with torch.fx is an experimental feature so that users do not think it is related to their setup being incompatible with transformers in general.

```suggestion
                f"{TORCH_FX_REQUIRED_VERSION} is supported for experimental torch.fx symbolic tracing feature."
```

Reply:
Indeed, thanks @michaelbenayoun. I'll commit this once I see the results of the torch GPU tests as re-committing would imply re-running all the tests on a doc change (whereas here I'll just commit & merge; then cancel the GPU tests related to this commit).